chloggen/internal/chlog/entry.go

```diff
@@ -47,6 +55,15 @@ var changeTypes = []string{
 	BugFix,
 }
 
+var classTypes = []string{
+	Receiver,
+	Exporter,
+	Processor,
+	Connector,
+	Extension,
+	Cmd,
+}
```

Comment:
This is tightly coupled to Collector. Should it not be configurable?

Reply:
This is indeed tied to the collector, all other uses can keep using the component field just like today.